/object_detector:
  ros__parameters:
    hsv_ranges:
    - 15.0
    - 20.0
    - 50.0
    - 200.0
    - 50.0
    - 150.0

